chase/frontend/components/dashboard/actions.tsx

```diff
@@ -0,0 +1,173 @@
+import React from "react";
+import WidgetTemplate from "./widgettemplate";
+import { Button } from "primereact/button";
+import { Dialog } from "primereact/dialog";
+import { InputText } from "primereact/inputtext";
+import { InputTextarea } from "primereact/inputtextarea";
+import { Dropdown } from "primereact/dropdown";
+
+import {
+  faComment,
+  faExclamationTriangle,
+  faGavel,
+  faPaperPlane,
+  faQuestionCircle,
+} from "@fortawesome/free-solid-svg-icons";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+
+export default function ActionsWidget() {
+  const [displayChairDialog, setDisplayChairDialog] = React.useState(false);
+  const [displayResearchDialog, setDisplayResearchDialog] =
+    React.useState(false);
+
+  const [category, setCategory] = React.useState("");
+  const categoryOption = [
+    { lable: "Gastrede anfragen", value: "gastrede", icon: faComment },
+    { lable: "Faktencheck", value: "faktencheck", icon: faExclamationTriangle },
+    {
+      lable: "Informationsanfrage",
+      value: "information",
+      icon: faQuestionCircle,
+    },
+    { lable: "Generalsekretär anfragen", value: "gs", icon: faGavel },
+    { lable: "Sonstiges", value: "sonstiges", icon: faPaperPlane },
+  ];
+  const [subjectLine, setSubjectLine] = React.useState("");
+  const [message, setMessage] = React.useState("");
+
+  const categoryOptionTemplate = (option) => {
+    return (
+      <div className="flex items-center gap-4">
+        <FontAwesomeIcon icon={option.icon} />
+        <span>{option.lable}</span>
+      </div>
+    );
+  };
+
+  const categoryOptionSelectedTemplate = (option, props) => {
+    if (option) {
+      return categoryOptionTemplate(option);
+    }
+    return props.placeholder;
+  };
+
+  const footerContent = ({ sendFunction }) => {
+    return (
+      <div>
+        <Button
+          label="Abbrechen"
+          icon="pi pi-times"
+          onClick={() => closeAndResetDialog()}
+          className="p-button-text"
+        />
+        <Button
+          label="Senden"
+          icon="pi pi-check"
+          onClick={() => sendFunction()}
+          autoFocus
+          disabled={subjectLine === "" || message === ""}
+        />
+      </div>
+    );
+  };
+
+  const closeAndResetDialog = () => {
```

Comment:
We should investigate if it makes sense/is possible to use forms here

Reply:
Thanks!

Reply:
We could also delay this until we've established an actual way of communicating with the backend and resolve this for now